templates/taxi/car_detail.html

```diff
@@ -1,7 +1,9 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})
+    <a style="float: right" href="{% url "taxi:car-update" pk=car.id %}">Update</a><a style="float: right" href="{% url 'taxi:car-delete' pk=car.id%}">Delete</a>
```

Comment:
Clean Code: Inline styles should be avoided. It's better to move all the styles to a separate CSS file. This will make your code cleaner and easier to maintain.

Reply:
done